.readthedocs.yml

```
@@ -0,0 +1,8 @@
+version: 2
+
+python:
+  version: '3.8'
+  install:
+  - requirements: requirements-docs.txt
+  - requirements: requirements-test.txt
+  - requirements: requirements.txt
```

```suggestion
  - requirements: requirements.txt
  - method: pip
    path: .
```

Documentation page for some class is empty:
https://etl-entities.readthedocs.io/en/latest/hwm/datetime_hwm.html

https://readthedocs.org/projects/etl-entities/builds/19389310/
```
WARNING: autodoc: failed to import class 'hwm.date_hwm.DateHWM' from module 'etl_entities'; the following exception was raised:
No module named 'etl_entities'
```